index.js

```diff
@@ -972,6 +972,22 @@
     return either instanceof Left ? l(either.value) : r(either.value);
   });
 
+  //# maybeToEither :: a -> Maybe b -> Either a b
+  //.
+  //. Converts a Maybe value to an Either value, using the supplied
+  //. parameter as the Left value if the Maybe is Nothing.
+  //.
+  //. ```javascript
+  //. > S.maybeToEither('Expecting an integer', S.parseInt(10, 'xyz'))
+  //. Left("Expecting an integer")
+  //.
+  //. > S.maybeToEither('Expecting an integer', S.parseInt(10, '42'))
+  //. Right(42)
+  //. ```
+  S.maybeToEither = def('maybeToEither', [a, Maybe], function(x, maybe) {
+    return maybe instanceof Nothing ? Left(x) : maybe.chain(Right);
```

Comment:
> Didn't realize I was allowed to reach into the `maybe`

We do so (in Sanctuary) so others don't need to. :)

> Interesting, since it doesn't have to return a `chain`able value at all, right (as when chaining with identity)?

In this case, though, `R.identity` *does* return a value of the same Chain (Maybe):

```javascript
> R.chain(R.identity, S.Just(S.Just(42)))
Just(42)
```